src/Main/ProjectFile/ProjectFileLoader.cpp

```diff
@@ -50,9 +50,7 @@ void ProjectFileLoader::registerComponent(ProjectFileComponent *comp) {
 void ProjectFileLoader::Load(const QString &filename) {
   if (filename.isEmpty()) return;
 
-  QString strTemp = QString("%1/%2%3").arg(Project::Instance()->projectPath(),
-                                           Project::Instance()->projectName(),
-                                           PROJECT_FILE_FORMAT);
+  QString strTemp = QString("/%1").arg(PROJECT_FILE_FORMAT);
```

Comment:
I think this should be reverted.

Reply:
But it's only storing "/.ospr" in strTemp.